src/app/feedback/feedback-view.component.html

```
@@ -1,6 +1,6 @@
 <!-- Feedback Template -->
 <mat-toolbar>
-  <button class="btnBack" mat-icon-button routerLink="/feedback"><mat-icon>arrow_back</mat-icon></button>
+  <button class="btnBack" mat-icon-button (click)="goBack()"><mat-icon>arrow_back</mat-icon></button>
```

HTMLHint (1.9.2)

[warning] The id and class attribute values must be in lowercase and split by a dash.

(id-class-value)